content/W2015/W15-event-coop.md

```diff
@@ -0,0 +1,19 @@
+Title: Co-op Showcase Panel
+Date: 2015-01-09 15:00
+Category: Events
+Slug: co-op-panel
```

Comment:
I would change this to "coop-panel" given that dashes are usually used as spaces in slugs.